### PR TITLE
Add Polish Documentation 

### PR DESCRIPTION
**Pull request by mbiesiad**
Hi. First of all, really great idea & project!

Here is Polish version 🔥
https://github.com/mbiesiad/gulp/tree/pl_PL/docs

Changes [here](https://github.com/gulpjs/gulp/pull/2456/files)

Best wishes,

### CODE REVIEW

**Comment by mbiesiad** [comment]
My pleasure! 🎉

**Comment by phated** [comment]
@mbiesiad we are moving away from linking to translations because we've had to remove multiple from our docs when the author abandoned them. Would you rather submit them directly in a PR?

**Comment by mbiesiad** [comment]
@phated good point. OK, sure. Stay in touch

**Comment by mbiesiad** [comment]
I agree with you, but could you send to me some link about recommendations for this? I created this in this way, because of current readme & how it looks for 'Simplified Chinese documentation' / 'Korean documentation'.
https://github.com/gulpjs/gulp/tree/master/docs

Do I should create ie. readme-pl.md, 1-quick-start-pl.md etc? Or just to create new folder for pl-PL?

Best wishes,

**Comment by phated** [comment]
@mbiesiad keeping everything in a pl_PL directory sounds good. Eventually our website will display the translations but docusaurus v2 doesn't have the new translation support working yet.

**Comment by mbiesiad** [comment]
@phated thank you for your reply & suggestions. I changed this a little bit, is it OK now?

**Comment by phated** [comment]
@mbiesiad this looks great! Are you planning to translate the rest of the documentation into Polish? How should we proceed?

**Comment by mbiesiad** [comment]
Thanks a lot! Yup, I hope so. Please, keep in touch

File: docs/locale/pl_PL/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+* [Dokumentacja język koreański][KoreanDocs] - gulp 한국어 참조 문서
+* [Dokumentacja język polski](https://github.com/mbiesiad/gulp/tree/pl_PL/docs) - gulp Dokumentacja
+
+
+## FAQ
+
+Zobacz [FAQ](FAQ.md) aby uzyskać odpowiedzi na najczęściej zadawane pytania.
+
+
+## Przepisy
```
Comment (jakubrpawlowski) [inline docs/locale/pl_PL/README.md:18]:
```suggestion
## Receptury
```

File: docs/locale/pl_PL/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Społeczność napisała [przepisy kulinarne](recipes#recipes) dla typowych przypadków użycia gulp.
```
Comment (jakubrpawlowski) [inline docs/locale/pl_PL/README.md:20]:
```suggestion
Społeczność napisała [receptury](recipes#recipes) dla typowych przypadków użycia gulp.
```

File: docs/locale/pl_PL/api/concepts.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+Chociaż Vinyl zapewnia sposób na opisanie pliku, potrzebny jest sposób na dostęp do tych plików. Każde źródło pliku jest dostępne za pomocą adaptera Vinyl.
+
+Adapter udostępnia:
+* Metoda z podpisem `src (globs, [opcje])` i zwraca strumień, który produkuje obiekty Vinyl.
+* Metoda z podpisem `dest (folder, [opcje])` i zwraca strumień, który zużywa obiekty Vinyl.
+* Wszelkie dodatkowe metody specyficzne dla ich nośnika wejścia / wyjścia - takie jak metoda `symlink`, którą zapewnia `winyl-fs`. Powinny zawsze zwracać strumienie, które produkują i / lub zużywają obiekty Vinyl.
+
+## Zadania
+
+Każde zadanie gulp to asynchroniczna funkcja JavaScript, która albo przyjmuje wywołanie zwrotne z pierwszym błędem, albo zwraca strumień, obietnicę, emiter zdarzeń, proces potomny lub observable. Z powodu pewnych ograniczeń platformy zadania synchroniczne nie są obsługiwane.
```
Comment (jakubrpawlowski) [inline docs/locale/pl_PL/api/concepts.md:29]:
```suggestion
Każde zadanie gulp to asynchroniczna funkcja JavaScript, która albo przyjmuje wywołanie zwrotne z pierwszym błędem, albo zwraca strumień, obietnicę, emiter zdarzeń, proces podrzędny lub observable. Z powodu pewnych ograniczeń platformy zadania synchroniczne nie są obsługiwane.
```

File: docs/locale/pl_PL/getting-started/2-javascript-and-gulpfiles.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+sidebar_label: JavaScript and Gulpfiles
+-->
+
+# JavaScript oraz Gulpfiles
+
+Gulp umożliwia wykorzystanie istniejącej wiedzy o języku JavaScript do pisania plików gulpfiles lub wykorzystanie doświadczenia z plikami gulpfiles do pisania zwykłego kodu JavaScript. Chociaż dostępnych jest kilka narzędzi upraszczających pracę z systemem plików i wierszem poleceń, wszystko, co piszesz, to czysty JavaScript.
+
+## Gulpfile wytłumaczony
+
+Plik gulpfile to plik w katalogu projektu o nazwie `gulpfile.js` (lub pisany wielkimi literami jako `Gulpfile.js`, jak Makefile), który automatycznie ładuje się po uruchomieniu polecenia `gulp`. W tym pliku często zobaczysz interfejsy API gulp, takie jak `src ()`, `dest ()`, `series ()` lub `parallel ()`, ale można użyć dowolnego waniliowego modułu JavaScript lub Node. Wszelkie wyeksportowane funkcje zostaną zarejestrowane w systemie zadań gulp.
```
Comment (jakubrpawlowski) [inline docs/locale/pl_PL/getting-started/2-javascript-and-gulpfiles.md:14]:
```suggestion
Plik gulpfile to plik w katalogu projektu o nazwie `gulpfile.js` (lub pisany wielkimi literami jako `Gulpfile.js`, jak Makefile), który automatycznie ładuje się po uruchomieniu polecenia `gulp`. W tym pliku często zobaczysz interfejsy API gulp, takie jak `src ()`, `dest ()`, `series ()` lub `parallel ()`, ale można użyć dowolnego standardowego modułu JavaScript lub Node. Wszelkie wyeksportowane funkcje zostaną zarejestrowane w systemie zadań gulp.
```

File: docs/locale/pl_PL/README.md (hunk 2 above)
Comment (mbiesiad) [reply docs/locale/pl_PL/README.md:20]: Ugh, I was obviously tired.. Changed, thanks!

**Comment by mbiesiad** [comment]
@jakubrpawlowski My pleasure. Thank you! Good points. Fixed, improved. 🔥

**Comment by phated** [comment]
Thank you both for this awesome work! I'm going to merge the other one since it includes these changes as well.

File: docs/locale/pl_PL/getting-started/1-quick-start.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
+<!-- front-matter
+id: quick-start
+title: Quick Start
+hide_title: true
+sidebar_label: Quick Start
+-->
+
+# Szybki start
```
Comment (phated) [inline docs/locale/pl_PL/getting-started/1-quick-start.md:8]:
```suggestion
# Szybki start
```

File: docs/locale/pl_PL/README.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+Cała dokumentacja objęta jest licencją CC0 *(rób co chcesz - domena publiczna)*.
+
+[![CC0](https://i.creativecommons.org/p/zero/1.0/88x31.png)](https://creativecommons.org/publicdomain/zero/1.0/)
+
+W zakresie, w jakim jest to możliwe na mocy prawa, [Fractal](http://wearefractal.com) zrzekł się wszelkich praw autorskich i pokrewnych lub powiązanych do tego dzieła.
+
+[SpanishDocs]: https://github.com/bucaran/gulp-docs-es
+[SimplifiedChineseDocs]: https://github.com/lisposter/gulp-docs-zh-cn
+[KoreanDocs]: https://github.com/preco21/gulp-docs-ko
+[PolishDocs]: https://github.com/mbiesiad/gulp/tree/pl_PL/docs
```
Comment (phated) [inline docs/locale/pl_PL/README.md:66]:
```suggestion
[PolishDocs]: /docs/locale/pl_PL/README.md
```

File: docs/locale/pl_PL/README.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
+# Dokumentacja gulp
+
+* [Pierwsze kroki](getting-started/) - Pierwsze kroki z gulp
+* [Dokumentacja API](api/) - Interfejs programowania, zdefiniowany
+* [Dokumentacja CLI](CLI.md) - Dowiedz się, jak wywoływać taski i korzystać z kompilatorów
+* [Pisanie wtyczki](writing-a-plugin/) - Podstawy pisania wtyczki gulp
+* [Czemu używać Pump?](why-use-pump/README.md) - Dlaczego używać modułu `pump` zamiast wywoływać `.pipe` yourself
+* [Dokumentacja język chiński uproszczony][SimplifiedChineseDocs] - gulp 简体中文文档
+* [Dokumentacja język koreański][KoreanDocs] - gulp 한국어 참조 문서
+* [Dokumentacja język polski](https://github.com/mbiesiad/gulp/tree/pl_PL/docs) - gulp Dokumentacja
```
Comment (phated) [inline docs/locale/pl_PL/README.md:10]:
```suggestion
* [Dokumentacja język polski][PolishDocs] - gulp Dokumentacja
```

File: docs/locale/pl_PL/api/concepts.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
+<!-- front-matter
+id: concepts
+title: API Concepts
+hide_title: true
+sidebar_label: Concepts
+-->
+
+# Pojęcia
```
Comment (phated) [inline docs/locale/pl_PL/api/concepts.md:8]:
```suggestion
# Pojęcia
```

File: docs/locale/pl_PL/getting-started/2-javascript-and-gulpfiles.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
+<!-- front-matter
+id: javascript-and-gulpfiles
+title: JavaScript and Gulpfiles
+hide_title: true
+sidebar_label: JavaScript and Gulpfiles
+-->
+
+# JavaScript oraz Gulpfiles
```
Comment (phated) [inline docs/locale/pl_PL/getting-started/2-javascript-and-gulpfiles.md:8]:
```suggestion
# JavaScript oraz Gulpfiles
```

File: docs/locale/pl_PL/README.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
+# Dokumentacja gulp
+
+* [Pierwsze kroki](getting-started/) - Pierwsze kroki z gulp
+* [Dokumentacja API](api/) - Interfejs programowania, zdefiniowany
```
Comment (phated) [inline docs/locale/pl_PL/README.md:4]:
```suggestion
* [Dokumentacja API](/docs/api/) - Interfejs programowania, zdefiniowany
```

File: docs/locale/pl_PL/README.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+* [Dokumentacja CLI](CLI.md) - Dowiedz się, jak wywoływać taski i korzystać z kompilatorów
```
Comment (phated) [inline docs/locale/pl_PL/README.md:5]:
```suggestion
* [Dokumentacja CLI](/docs/CLI.md) - Dowiedz się, jak wywoływać taski i korzystać z kompilatorów
```

File: docs/locale/pl_PL/README.md [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+* [Pisanie wtyczki](writing-a-plugin/) - Podstawy pisania wtyczki gulp
```
Comment (phated) [inline docs/locale/pl_PL/README.md:6]:
```suggestion
* [Pisanie wtyczki](/docs/writing-a-plugin/) - Podstawy pisania wtyczki gulp
```

File: docs/locale/pl_PL/README.md [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+* [Czemu używać Pump?](why-use-pump/README.md) - Dlaczego używać modułu `pump` zamiast wywoływać `.pipe` yourself
```
Comment (phated) [inline docs/locale/pl_PL/README.md:7]:
```suggestion
* [Czemu używać Pump?](/docs/why-use-pump/README.md) - Dlaczego używać modułu `pump` zamiast wywoływać `.pipe` yourself
```

File: docs/locale/pl_PL/README.md [hunk 14]
```diff
@@ -0,0 +1,66 @@
...
+* [Dokumentacja język chiński uproszczony][SimplifiedChineseDocs] - gulp 简体中文文档
+* [Dokumentacja język koreański][KoreanDocs] - gulp 한국어 참조 문서
+* [Dokumentacja język polski](https://github.com/mbiesiad/gulp/tree/pl_PL/docs) - gulp Dokumentacja
+
+
+## FAQ
+
+Zobacz [FAQ](FAQ.md) aby uzyskać odpowiedzi na najczęściej zadawane pytania.
```
Comment (phated) [inline docs/locale/pl_PL/README.md:15]:
```suggestion
Zobacz [FAQ](/docs/FAQ.md) aby uzyskać odpowiedzi na najczęściej zadawane pytania.
```

File: docs/locale/pl_PL/README.md [hunk 15]
```diff
@@ -0,0 +1,66 @@
...
+
+
+## Receptury
+
+Społeczność napisała [receptury](recipes#recipes) dla typowych przypadków użycia gulp.
```
Comment (phated) [inline docs/locale/pl_PL/README.md:20]:
```suggestion
Społeczność napisała [receptury](/docs/recipes#recipes) dla typowych przypadków użycia gulp.
```

**Pull request merged**